pkg/binding/backend/flags.go

```diff
@@ -59,6 +60,13 @@ func (o *FlagsBackend) SelectWorkloadInstance(workloadName string) (string, sche
 	return "", schema.GroupVersionKind{}, fmt.Errorf("group/kind %q not found on the cluster", selectedGroup+"/"+selectedKind)
 }
 
+// SelectNamespaceForServiceInstances returns the namespace which services instances should be listed from.
+// An empty return value means that service instances will be listed from the current namespace.
+// Otherwise, they will be listed from the specified namespace.
```

Comment:
Same request as for the file `interactive.go` to keep comment at one place and add only implementation specific comment where an interface method is getting implemented.